screening/safe-you.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "Safe YOU",
+  "evidenceURL5": "https://www.forbes.com/sites/jackieabramian/2021/04/01/safe-you-mobile-app-shields-women-against-domestic-violence/?sh=3b7c82592e73",
```

Comment:
@nathanbaleeta As you see there is some sort of bug with the submission form, this field should not appear here

Reply:
@lacabra it could be resulting from a memory leak. Have observed it happening a couple of times.